client/src/components/NewReviewForm.js

```diff
@@ -5,22 +5,29 @@ import translateServerErrors from "../services/translateServerErrors"
 const NewReviewForm = (props) => {
   const [newReview, setNewReview] = useState({
     body: "",
-    score: ""
+    score: "",
+    userId: "",
+    votes: ""
   })
 
   const [errors, setErrors] = useState([])
 
+  const userId = props.userId
+
   const handleInputChange = (event) => {
     setNewReview({
       ...newReview,
-      [event.currentTarget.name]: event.currentTarget.value
+      [event.currentTarget.name]: event.currentTarget.value,
+      userId: userId,
+      votes: []
```

Comment:
A new review shouldn't need to be created with votes - I think we briefly chatted about this yesterday, there were errors because after the review posts it tries to map over undefined votes (I think it was mentioned this would be resolved in a future story?) however, I would recommend removing instances of `votes` in this component, and instead if the `map` is failing, you can add a `?` for conditional chaining when you map over the votes that checks if the property exists

```js
const votes = review.votes?.map(...)
```